hb_report/utillib.py

```diff
@@ -294,6 +294,12 @@ def collect_info():
         if l == constants.HA_LOG and l != constants.HALOG_F:
             os.symlink(constants.HALOG_F, os.path.join(constants.WORKDIR, os.path.basename(l)))
             continue
+        if is_our_log(l, constants.FROM_TIME, constants.TO_TIME) == 4:
+            log_warning("found irregular log file %s" % l)
```

Comment:
Should this really be a warning? Maybe info is more appropriate. After all we expect ha-cluster-bootstrap.log to be irregular, for example.